### PR TITLE
Document how to apply the source available license

### PR DESCRIPTION
**Pull request by lieut-data**
#### Summary
image
image
image

### CODE REVIEW

File: site/content/extend/plugins/source-available-license.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+title: "Source Available License"
+subsection: Plugins (Beta)
+weight: 91
+---
+
+Some plugins authored by Mattermost are licensed under the [Mattermost Source Available License](https://docs.mattermost.com/overview/faq.html#mattermost-source-available-license). This document outlines how to apply the license in various situations.
+
+## How do I apply the license to an enterprise-only plugin?
+
+An enterprise-only plugin is a plugin that requires a valid Mattermost Enterprise E20 license. It is not designed to be used with Team Edition or any other enterprise license.
```
Comment (justinegeffen) [inline site/content/extend/plugins/source-available-license.md:11]:
```suggestion
An Enterprise-only plugin is a plugin that requires a valid Mattermost Enterprise E20 license. It is not designed to be used with Team Edition or any other Enterprise license.
```

File: site/content/extend/plugins/source-available-license.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+---
+title: "Source Available License"
+subsection: Plugins (Beta)
+weight: 91
+---
+
+Some plugins authored by Mattermost are licensed under the [Mattermost Source Available License](https://docs.mattermost.com/overview/faq.html#mattermost-source-available-license). This document outlines how to apply the license in various situations.
+
+## How do I apply the license to an enterprise-only plugin?
```
Comment (justinegeffen) [inline site/content/extend/plugins/source-available-license.md:9]:
```suggestion
## How do I apply the license to an Enterprise-only plugin?
```

File: site/content/extend/plugins/source-available-license.md (hunk 1 above)
Comment (justinegeffen) [reply site/content/extend/plugins/source-available-license.md:11]: Note on this: I'll need to review the other documentation to make sure enterprise/Enterprise is presented correctly so there might be inconsistencies in that right now.

File: site/content/extend/plugins/source-available-license.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+```md
+## License
+
+This repository is licensed under the [Mattermost Source Available License](LICENSE) and requires a valid Enterprise E20 license. See [Mattermost Source Available License](https://docs.mattermost.com/overview/faq.html#mattermost-source-available-license) to learn more.
+```
+
+## How do I apply the license to a mixed-license plugin?
+
+An mixed-license plugin is a plugin some of whose components require a valid Mattermost Enterprise E20 license. Not all features are available when used with Team Edition or any other enterprise license.
```
Comment (justinegeffen) [inline site/content/extend/plugins/source-available-license.md:24]:
```suggestion
An mixed-license plugin includes components that require a valid Mattermost Enterprise E20 license. Not all features are available when used with Team Edition or any other enterprise license.
```

File: site/content/extend/plugins/source-available-license.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+1. Organize the enterprise-only, server-side parts of your plugin in a dedicated folder, typically `server/enterprise`.
```
Comment (justinegeffen) [inline site/content/extend/plugins/source-available-license.md:26]:
```suggestion
1. Organize the Enterprise-only, server-side parts of your plugin in a dedicated folder, typically `server/enterprise`.
```

**Comment by lieut-data** [comment]
Aweseome -- thanks, @justinegeffen!!

**Comment by lieut-data** [comment]
Perfect: thanks @jfrerich!

**Comment by lieut-data** [comment]
@aaronrothschild, given this is already live, I'm going to go ahead and merge and we can follow-up with any necessary changes later.

**Pull request merged**